CONTRIBUTING.md

```diff
@@ -68,6 +78,18 @@ scripts/format.sh
 - you must update the documentation
 - you must run the above scripts to format and line
 
+## Code development process
+
+1. Fork and clone repository
+2. Create a new virtual environment with conda/virtualenv
+3. Activate environment and install earthaccess as editable source
+    1. `pip install -e .`
+    2. `poetry install` (note, if we use Poetry with conda we need to tell poetry to not use its own virtual environments)
+4. export EDL env variables (EARTHDATA_USER, EARTHDATA_PASSWORD)
+   `export EARTHDATA_USER=foo`
+   `export EARTHDATA_PASSWORD=bar`
```

Comment:
I don't think this is necessary. I'm able to run all tests without setting these, so I'm not sure what would require these otherwise.

Reply:
Yeah, I noticed post-PR that if you have your .netrc set up then this is not necessary. I will update accordingly.

Reply:
This credentials are only needed for the integration tests 
* Pick 2 random collections from 5 DAACs (NSIDC, GES-DISC, PODAAC, LP, ORNL)
* Test that we can download and open a granule form these collections, they failed a lot due custom endpoints EULAs etc so the tests pass if we have 90% success. We need to make this more deterministic (perhaps with the "test the top 10 for each DAAC" approach) @doug-newman-nasa

Reply:
There _was_ one unit test that does a CMR request and I think that is looking for credentials `test_data_link`